### PR TITLE
Finish ex7

### PR DESCRIPTION
**Pull request by Exg020**

### CODE REVIEW

File: src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student5/BookController5.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package com.xpanxion.java.springboot.da1.demo.controller.student5;
-import com.xpanxion.java.springboot.da1.demo.model.student5.Book;
+import com.xpanxion.java.springboot.da1.demo.model.student4.student5.Book;
```
Comment (mburolla) [inline src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student5/BookController5.java]: `student4.student5` looks odd.  This should be just student5.

File: src/main/java/com/xpanxion/java/springboot/da1/demo/model/student4/student5/Book.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.xpanxion.java.springboot.da1.demo.model.student5;
+package com.xpanxion.java.springboot.da1.demo.model.student4.student5;
```
Comment (mburolla) [inline src/main/java/com/xpanxion/java/springboot/da1/demo/model/student4/student5/Book.java]: student 4?

Comment (Exg020) [reply src/main/java/com/xpanxion/java/springboot/da1/demo/model/student4/student5/Book.java]: i have no idea on why that was changed

Comment (Exg020) [reply src/main/java/com/xpanxion/java/springboot/da1/demo/model/student4/student5/Book.java]: i see what happened fixed it

**Pull request merged**